script/build.go

```diff
@@ -50,7 +51,12 @@ func mainBuild() {
 	cmd, _ := exec.Command("git", "rev-parse", "--short", "HEAD").Output()
 
 	if len(cmd) > 0 {
-		LdFlag = strings.TrimSpace("-X github.com/git-lfs/git-lfs/config.GitCommit=" + string(cmd))
+		LdFlags = append(LdFlags, "-X", strings.TrimSpace(
+			"github.com/git-lfs/git-lfs/config.GitCommit="+string(cmd),
+		))
+	}
+	if !*BuildDwarf || *BuildAll {
+		LdFlags = append(LdFlags, "-s", "-w")
```

Comment:
Ah, this is an artifact of when I was playing around with the command before I pushed the PR. At this stage, the flag was `--no-dwarf`, so the absence of it would _include_ the DWARF tables by default. I wanted `--all` to be the equivalent of `--all --no-dwarf` without having to type that every time.

Since the flag is now `--dwarf`, I went ahead and removed that implicit behavior in 5f18308. Thanks for pointing this out!